system-tests/004/fs.test.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const stylelint = require('../../lib');
+const { caseConfigFile, caseFilesForFix, prepForSnapshot } = require('../systemTestUtils');
+
+const CASE_NUMBER = '004';
+
+it('no-fs - errored state for reportNeedlessDisables', async () => {
```

Comment:
Should these tests start with `fs -` rather than `no-fs -`?

Reply:
thanks, fixed